packages/frameworks/test/frameworks.unit.test.ts

```diff
@@ -54,7 +53,6 @@ const Schema = {
   items: {
     type: 'object',
     required: ['name', 'slug', 'logo', 'description', 'settings'],
```

Comment:
Let's add the additional required props to this test

Reply:
I'll add the string ones, but there's no way to test functions with `ajv` so maybe we can manually check for that after ajv.